### DISCUSSION TITLE
Application Feature ⚙️ list 📃

### DISCUSSION THREAD

**Discussion by adishmuhammed** [VC-AndroidDevelopers/SecureCred#9] [Ideas]
🙏 Please share your 🤔 ideas on the essential 📱 app features!

Here are some features we've discussed:
- 💾 Store password and 💳 card information
- 🔒 Encrypt passwords 📍 locally
- 🤝 Share credentials with others
- 🔄 Synchronize credentials across multiple devices

**Comment by ShabeerAlavi-Codez** [comment]
> pray Please share your thinking ideas on the essential iphone app features!
> 
> Here are some features we've discussed:
> 
> * floppy_disk Store password and credit_card card information
> * lock Encrypt passwords round_pushpin locally
> * handshake Share credentials with others
> * arrows_counterclockwise Synchronize credentials across multiple devices

✅Password assessment report
✅log every credentials request
✅Password generator

**Comment by adishmuhammed** [comment]
🤔 Should we  add an option to add 2 step verification code also?

**Reply by ShabeerAlavi-Codez** [reply]
Yes

**Comment by adishmuhammed** [comment]
Get password 🔑 form other password managers like Google, Microsoft and others
